robot_radius: 0.5
coverage_radius: 0.5